docs/install/installEverest.md

```diff
@@ -0,0 +1,98 @@
+# Install Everest
+
+## Setup
+
+Percona Everest has two primary components:
+
+- [Percona CLI](https://github.com/percona/percona-everest-cli), which connects Percona Everest to your Kubernetes cluster.
+- Everest application (UI for simplicity), which you can spin up using the instructions below.
+
+### Supported operators
+
+- Percona Operator for MySQL Based on Percona XtraDB Cluster (PXC) 1.12
+- Percona Operator for MongoDB (PSMDB) 1.14
+- Percona Operator for PostgreSQL (PG) 2.2
+
+## Before you start
+
+Percona Everest assists with installing all the necessary operators and required packages, but does not currently help with spinning up a Kubernetes cluster.
+Therefore, before installing Everest, ensure that you have a publicly accessible Kubernetes cluster.
+
+We recommend [setting one up on the Amazon Elastic Kubernetes Service (EKS)](../install/eks.md) or on [Google Kubernetes Engine (GKE)](../install/GKE-k8s-cluster), as it may be difficult to currently spin one up on local Kubernetes installations (minikube, kind, k3d, or similar products).
+
+## Prerequisites
+
+Before getting started with Percona Everest, we recommend that you:
+
+1. Install [Docker](https://docs.docker.com/engine/install/) and its compose plugin (included by default in the aforementioned docker engine installation instructions).
+2. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) for EKS or the [gcloud CLI](https://cloud.google.com/sdk/docs/install) for GKE.
+3. Download the kubeconfig file from the Kubernetes cluster that you want to use with Everest. By default, Everest uses the file available under *~/.kube/config.*
+
+## Get started
+
+To install and provision Percona Everest:
+
+1. Download the Docker compose file:
+```
+curl https://raw.githubusercontent.com/percona/percona-everest-backend/v0.3.0/deploy/quickstart-compose.yml -o quickstart-compose.yml
+```
+2. Initialize the Everest container and its internal PostgreSQL database by executing one of the following commands, each offering distinct access configurations:
+
+    - to limit access to the localhost interface (default):
+    ```sh
+    docker compose -f quickstart-compose.yml up -d
+    ```
+    - to expose Everest though a specific interface, use that interface's IP address: 
+    ```sh
+    EVEREST_BIND_ADDR=<ip address>; docker compose -f quickstart-compose.yml up -d
+    ```
+    - to allow access from any interface on the host machine:
+    ```sh
+    EVEREST_BIND_ADDR=0.0.0.0; docker compose -f quickstart-compose.yml up -d
```

Comment:
This is not secure. I think there was some PR from IT which mentioned that we should not have 0.0.0.0 in quickstart.yml

Reply:
Thanks Manish, we added the `(not recommended)` to make it clear that the users should proceed at their own risk.

Reply:
Actually what the security team complained about was having 0.0.0.0 as the default which we fixed. Now the default is 127.0.0.1 which is safe but we still allow the users to change it if they need to.